ceph/nvmeof/nvme_cli.py

```diff
@@ -0,0 +1,61 @@
+from ceph.ceph_admin.common import config_dict_to_string
+from cli import Cli
+
+
+class NVMeCLI(Cli):
+    """NVMe CLI commands.
+
+    - Need to initialize Cli class with Ceph node object.
+    - NQN: NVMe qualified name.
+    """
+
+    def configure(self):
+        """Install NVME CLI."""
+        configure_cmds = [
+            ("yum install -y nvme-cli fio", True),
+            ("modprobe nvme-fabrics", True),
+        ]
+        for cmd in configure_cmds:
+            self.execute(*cmd)
+
+    def discover(self, **kwargs):
+        """Discover the subsystems.
+
+        Example::
+
+            kwargs:
+                transport: tcp                  # Transport protocol
+                traddr: IP address              # Transport address
+                trsvcid: Transport port number  # Transport port number
+                output-format: json
+        """
+        return self.execute(
+            cmd=f"nvme discover {config_dict_to_string(kwargs)}",
```

Comment:
IMO, It would be good to log output of these commands - discover, connect and list

Reply:
Updated.